### PR TITLE
Grouped Guildmates Indicator

### PR DESCRIPTION
**Pull request by coliyeah**
As previously mentioned, adds a column to show guildmates who are grouped with the player.  I'm new to git and github, but hopefully I've gotten it done correctly. =)

### CODE REVIEW

File: picoGuild.lua [hunk 1]
```diff
@@ -137,8 +137,9 @@ function dataobj.OnEnter(self)
 				elseif level > (mylevel + 5) then lr, lg, lb = 1, 0, 0 end
 				if area == myarea then ar, ag, ab = 0, 1, 0 end
 				local levelcolor = (level >= (mylevel - 5) and level <= (mylevel + 5)) and "|cff00ff00" or ""
-				tip:AddMultiLine((level < 10 and "0" or "")..level, name, area or "???", note, officernote, rank,
-					lr,lg,lb, cc.r,cc.g,cc.b, ar,ag,ab, nil,nil,nil, 1,1,0, .7,.7,1)
+				local grouped =((UnitInParty(name) or UnitInRaid(name)) and true) and "@" or ""
```
Comment (tekkub) [inline picoGuild.lua:140]: `and true`?

**Comment by tekkub** [comment]
So I'm pondering on this... I don't like adding columns to the tooltips, they quickly get out of hand.  Perhaps appending an indicator to the end of the name?

```
Tekkub [P]
Joebob [R]
```

File: picoGuild.lua (hunk 1 above)
Comment (coliyeah) [inline picoGuild.lua:140]: Sorry, legacy code from when UnitInParty and UnitInRaid worked differently (I think).  Seems it can be dropped now.

**Comment by coliyeah** [comment]
Personally I prefer the separate column in this instance.  It will only ever be one character wide (plus padding).

If you simply append a character to the name I'd think it will either a)  be the same color as the name, or b)  cause issues trying to have two colors in one column.  Having the same color as the name makes it stand out far less, so that takes the easy route out of the equation for me.

**Pull request merged**